<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>registerwindow</class>
 <widget class="QDialog" name="registerwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>815</width>
    <height>688</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="user_label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt; font-weight:700;&quot;&gt;USERNAME&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="pass_label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt; font-weight:700;&quot;&gt;PASSWORD&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="confirm_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt; font-weight:700;&quot;&gt;CONFIRM PASS&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="user_line">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>30</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="pass_line">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>80</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="confirm_line">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>120</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="date_label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>200</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt; font-weight:700;&quot;&gt;DATE OF BIRTH&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QComboBox" name="month_box">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>103</width>
     <height>32</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>MONTH</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>jan</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>feb</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>mar</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>april</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>may</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>june</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>july</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>august</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>sep</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>nov</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>dec</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="date_line">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>260</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="year_line">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>260</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>230</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:700;&quot;&gt;DAY&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>230</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:700;&quot;&gt;YEAR&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="gender_label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>320</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt; font-weight:700;&quot;&gt;GENDER&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="gender_box">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>360</y>
     <width>121</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>CHOOSE</string>
   </property>
   <widget class="QRadioButton" name="male_but">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>99</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>MALE</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="female_but">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>99</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>FEMALE</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="type_label">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>320</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt; font-weight:700;&quot;&gt;ACCOUNT TYPE&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="type_box">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>360</y>
     <width>120</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>TYPE</string>
   </property>
   <widget class="QRadioButton" name="user_but">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>99</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>USER</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="admin_but">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>99</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>ADMIN</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="genre_box">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>350</y>
     <width>251</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>GENRES</string>
   </property>
   <widget class="QCheckBox" name="action_but">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>85</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>ACTION</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="comedy_but">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>70</y>
      <width>85</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>COMEDY</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="horror_but">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>110</y>
      <width>85</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>HORROR</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="romance_but">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>85</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>ROMANCE</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="drama_but">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>70</y>
      <width>85</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>DRAMA</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="other_but">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>110</y>
      <width>85</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>OTHER...</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="fav_label">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>310</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt; font-weight:700;&quot;&gt;FAVOURITE GENRES&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="register_but2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>540</y>
     <width>171</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>REGISTER</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
